### PR TITLE
Add File Download Logs meta table, and API Request Logs meta table.

### PR DESCRIPTION
**Pull request by mintplugins**
#7285

### CODE REVIEW

**Comment by cklosowski** [comment]
@JJJ would you take a quick look over this and make sure that we're doing these new meta tables the right way, as you intended with the BerlinDB project?

File: includes/logs/api-request-log/functions.php [hunk 1]
```diff
@@ -119,86 +109,82 @@ function edd_update_api_request_log( $api_request_log_id = 0, $data = array() )
...
  *
- * @return EDD\Api_Request_Logs\Api_Request_Log|false Log object if successful, false otherwise.
+ * @return \EDD\Logs\Api_Request_Log|false Api_Request_Log object if successful,
+ *                                         false otherwise.
  */
 function edd_get_api_request_log_by( $field = '', $value = '' ) {
 	$api_request_logs = new EDD\Database\Queries\Log_Api_Request();
 
-	// Return log.
+	// Return note
```
Comment (JJJ) [inline includes/logs/api-request-log/functions.php]: This looks accidental.

**Comment by JJJ** [comment]
Check list:
- [ ] the tables were created correctly 
- [ ] without any errors or problems
- [ ] code reviewed with 1 code comment that looks accidental
- [ ] did y'all settle on final names for these tables (they are still `log_` instead of `logs_`?)

**Comment by cklosowski** [comment]
I believe they should be `logs_` the `plural` -> `singular<meta>` is used for the last part of the table name. The current meta example of `edd_logs` and `edd_logmeta` is unique in that the `logs` are the plural to `logmeta` singular.

**Comment by JJJ** [comment]
We _could_ go as far as renaming `wp_edd_logs` to `wp_edd_logs_general` if we have concerns about being the database table names not being explicit enough. We had talked about it in the very beginning and decided against it, as it's somewhat implied by omitting it.

**Comment by mintplugins** [comment]
@cklosowski @JJJ Just so I'm sure we are all on the same page, what do we think the names of each logging and meta table should be?

1. General Logs Table: 
2. General Logs Meta Table: 
3. API Request Logs Table: 
4. API Request Logs Meta Table: 
5. API Request Logs Table: 
6. File Download Logs Meta Table:

**Comment by mintplugins** [comment]
General Logs Table: edd_logs
General Logs Meta Table: edd_logmeta
API Request Logs Table: edd_logs_api_requests
API Request Logs Meta Table: edd_logs_apirequestmeta
File Download Logs Table: edd_logs_file_downloads
File Download Logs Meta Table: edd_logs_file_downloadmeta

**Comment by mintplugins** [comment]
✅Tables renamed
✅Relevant unit tests passing
✅Manually confirmed tables are correctly created

**Pull request merged**